src/Compiler/Checking/MethodCalls.fsi

```diff
@@ -83,7 +83,7 @@ type AssignedCalledArg<'T> =
 
 /// Represents the possibilities for a named-setter argument (a property, field, or a record field setter)
 type AssignedItemSetterTarget =
-    | AssignedPropSetter of PropInfo * MethInfo * TypeInst
+    | AssignedPropSetter of staticTyOpt: TType option * pinfo: PropInfo * minfo: MethInfo * pminst: TypeInst
```

Comment:
What does pminst stand for?

Reply:
`pm` = parameters, if I remember correctly